src/vm/analysis/type_checker/tests/mod.rs

```diff
@@ -12,7 +12,9 @@ use vm::types::{Value, PrincipalData, TypeSignature, FunctionType, FixedFunction
                 QualifiedContractIdentifier};
 
 use vm::database::MemoryBackingStore;
-use vm::types::TypeSignature::{IntType, BoolType, BufferType, UIntType, PrincipalType};
+use vm::types::TypeSignature::{IntType, BoolType, SequenceType, UIntType, PrincipalType};
+use vm::types::{SequenceSubtype::*, StringSubtype::*};
+
```

Comment:
I notice all the tests in this file got changed from `(buff ..)` to `(string-ascii ..)`.  Are the `(buff ..)` tests preserved somewhere?